install_config/persistent_storage/topics/glusterfs_example_full.adoc

```diff
@@ -0,0 +1,106 @@
+----
+[OSEv3:vars]
+...
+openshift_hosted_registry_storage_kind=glusterfs
+
+openshift_storage_glusterfs_block_deploy=false
+
+openshift_storage_glusterfs_registry_block_deploy=true
+openshift_storage_glusterfs_registry_block_host_vol_size=50
+openshift_storage_glusterfs_registry_block_storageclass=true
+openshift_storage_glusterfs_registry_block_storageclass_default=true
+
+openshift_storageclass_default=false
+----
+
+. Add `glusterfs` and `glusterfs_registry` in the `[OSEv3:children]` section to
+enable the `[glusterfs]` and `[glusterfs_registry]` groups:
++
+----
+[OSEv3:children]
+masters
+nodes
+glusterfs
+glusterfs_registry
+----
+
+. Add `[glusterfs]` and `[glusterfs_registry]` sections with entries for each
+storage node that will host the GlusterFS storage. For each node, set
+`glusterfs_devices` to a list of raw block devices that will be completely
+managed as part of a GlusterFS cluster. There must be at least one device
+listed. Each device must be bare, with no partitions or LVM PVs. Specifying the
+variable takes the form:
++
+----
+<hostname_or_ip> glusterfs_devices='[ "</path/to/device1/>", "</path/to/device2>", ... ]'
+----
++
+For example:
++
+----
+[glusterfs]
+node11.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node12.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node13.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+
+[glusterfs_registry]
+node14.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node15.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node16.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+----
+
+. Add the hosts listed under `[glusterfs]` and `[glusterfs_registry]` to the
+`[nodes]` group:
++
+----
+[nodes]
+...
+node11.example.com openshift_schedulable=True openshift_node_labels="{'role': 'app'}"
+node12.example.com openshift_schedulable=True openshift_node_labels="{'role': 'app'}"
+node13.example.com openshift_schedulable=True openshift_node_labels="{'role': 'app'}"
+node14.example.com openshift_schedulable=True openshift_node_labels="{'role': 'infra'}"
+node15.example.com openshift_schedulable=True openshift_node_labels="{'role': 'infra'}"
+node16.example.com openshift_schedulable=True openshift_node_labels="{'role': 'infra'}"
+----
+
+. Run the Advanced Installer without deploying Logging or Metrics. This can be
+either greenfield:
++
+----
+ansible-playbook -i <path_to_inventory_file> /usr/share/ansible/openshift-ansible/playbooks/prerequisites.yml
+
+ansible-playbook -i <path_to_inventory_file> /usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml
+----
++
+or brownfield:
++
+----
+ansible-playbook -i <path_to_inventory_file> /usr/share/ansible/openshift-ansible/playbooks/openshift-glusterfs/config.yml
+----
+
+. Add the following variables under `[OSEv3:vars]`:
++
+----
+[OSEv3:vars]
+...
+openshift_metrics_storage_kind=dynamic
+openshift_logging_storage_kind=dynamic
+openshift_logging_es_pvc_size=10Gi
```

Comment:
I would still add this to minimum options in this section:

openshift_hosted_metrics_storage_volume_size=<default_size>
openshift_hosted_metrics_deploy=true
openshift_hosted_logging_deploy=true

Reply:
Those are all defaults so there's no need to specify them for this example.